motionplan/tpSpaceRRT.go

```diff
@@ -657,7 +662,7 @@ func (mp *tpSpaceRRTMotionPlanner) attemptExtension(
 		endNode := reseedCandidate.newNodes[len(reseedCandidate.newNodes)-1]
 		distTravelledByCandidate := 0.
 		for _, newNode := range reseedCandidate.newNodes {
-			distTravelledByCandidate += newNode.Q()[2].Value
+			distTravelledByCandidate += (newNode.Q()[3].Value - newNode.Q()[2].Value)
```

Comment:
I am curious about the comment you left on negative distance values.
Would an example be something like `[index, alpha, -15, -30]`?
Then `distTravelledByCandidate = -15`. Do we need to do an `math.Abs()` here?

Reply:
This area is not done yet, not sure yet how this will work.

Reply:
Distances here will always be positive in the current state; goal tree inputs are flipped later.

However I have added the `Abs` as this may change in the future.